workers/datasets_based/src/datasets_based/workers/first_rows.py

```diff
@@ -535,9 +535,12 @@ def compute_first_rows_response(
             use_auth_token=use_auth_token,
         )
     except Exception as err:
-        if max_size_fallback:
-            warnings.warn(<A WARNING MESSAGE HERE>)
         MAX_SIZE_FALLBACK = 100_000_000
+        if max_size_fallback:
+            logging.warning(
+                f"The parameter 'max_size_fallback' is deprecated. The hard-coded value `{MAX_SIZE_FALLBACK}`"
+                " will be used instead."
+            )
```

Comment:
Do you prefer logging rather than `warnings.warn("MESSAGE", FutureWarning)`?

Reply:
oh, I thought it was the same.

Reply:
Not exactly... https://docs.python.org/3/howto/logging.html#when-to-use-logging

Reply:
> Issue a warning regarding a particular runtime event
> - warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning
> - logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted

Reply:
just discovered https://docs.python.org/3/library/warnings.html 😅